src/modules/libraryModule.js

```diff
@@ -14,80 +14,127 @@ export default class Library {
 
   removeButton;
 
+  inputField;
+
+  checkboxButton;
+
   static pushToStorage = (obj) => {
     const stringify = JSON.stringify(obj);
-    sessionStorage.setItem('strShelf', stringify);
+    localStorage.setItem('strShelf', stringify);
   }
 
   static displayBook(title, id) {
     return `
-        <div class="box unchecked"></div>
-        <div class="task title-author">${title}</div>
-        <div class="handle removeButton" id="${id}" type="button"></div>
+    <div class="box unchecked"></div>
+    <input class="task" type="text" value="${title}" name="Item1">
+    <div class="handle removeButton" id="${id}" type="button"></div>
+        `;
+  }
+
+  static displayCheckedBook(title, id) {
+    return `
+      <div class="box checked"></div>
+      <input class="task" type="text" value="${title}" name="Item1">
+      <div class="handle removeButton" id="${id}" type="button"></div>
         `;
   }
 
   pullFromStorage = () => {
-    const parsed = JSON.parse(sessionStorage.getItem('strShelf'));
+    const parsed = JSON.parse(localStorage.getItem('strShelf'));
     let counter = 1;
     const preShelf = [];
-    for (let i = 0; i < this.shelf.length; i += 1) {
+    // Returns the length of the localstorage shelf
+    // (even when the page reloads)
+    const shelfSize = Object.values(parsed).length;
+
+    for (let i = 0; i < shelfSize; i += 1) {
+      // Returns the Nth item saved on the localstorage
+      if (this.shelf.length === 0) {
+        for (let i = 0; i < shelfSize; i += 1) {
+          this.shelf.push(Object.values(parsed)[i]);
+        }
+      }
+      // Get the completed value of parsed book
       const parsedBook = parsed[`${counter}`];
-      const tempShelf = Library.displayBook(parsedBook.title, counter);
-      preShelf.unshift(tempShelf);
+      let tempShelf = Library.displayBook(parsedBook.title, counter);
+      if (parsedBook.completed) {
+        tempShelf = Library.displayCheckedBook(parsedBook.title, counter);
+      }
+      preShelf.push(tempShelf);
       counter += 1;
     }
+    return preShelf;
+  }
 
```

Comment:
- [ ] It would be better if there is a default todo list so we can interact with

Reply:
I'm not sure what it means, but I'm so close to the deadline, I promise I'll do it in the next PR (I have less than 2 hours to finish the weeks projects)